src/current/_includes/molt/replicator-flags.md

```diff
@@ -39,7 +39,7 @@
 | `--targetJitterTime`                   | `DURATION` | The time over which to jitter database pool disconnections.<br><br>**Default:** `15s`                                                                                                                                                                                                                                           |
 | `--targetMaxLifetime`                  | `DURATION` | The maximum lifetime of a database connection.<br><br>**Default:** `5m0s`                                                                                                                                                                                                                                                       |
 | `--targetMaxPoolSize`                  | `INT`      | The maximum number of target database connections.<br><br>**Default:** `128`                                                                                                                                                                                                                                                    |
-| `--targetSchema`                       | `STRING`   | The SQL database schema in the target cluster to update.                                                                                                                                                                                                                                                                        |
+| `--targetSchema`                       | `STRING`   | The SQL database schema in the target cluster to update. CockroachDB schema names must be fully qualified in the format `database.schema`.                                                                                                                                                                                      |
```

Comment:
Looking at this now, i don't think this should be part of Global Flags since its really only for X2C